production/helm/promtail/values.yaml

```diff
@@ -309,3 +309,51 @@ config:
       - __meta_kubernetes_pod_uid
       - __meta_kubernetes_pod_container_name
       target_label: __path__
+  - entry_parser: '{{ .Values.entryParser }}'
+    job_name: kubernetes-pods-static
```

Comment:
add comment about which version match is better.

Reply:
I don't know what you mean

Reply:
Add comment about that job_name match which k8s version, better to maintain in the future.

Reply:
This has nothing to do with what k8s version applies. This PR and all job_names in the scrape config apply to all k8s versions. I think you might be confused that this PR is a fix for the CRI 1.14 issue, but it's instead to fix the control pod scraping.